### PR TITLE
Recover active tab in run details page after reload

### PR DESCRIPTION
**Pull request by roubkar**
Added routing for run details page tabs.

### CODE REVIEW

**Comment by roubkar** [comment]
Resolves https://github.com/aimhubio/aim/issues/1150

**Review by rubenaprikyan — APPROVED** [review]
lgtm!

File: aim/web/ui/src/pages/RunDetail/RunDetail.tsx [hunk 1]
```diff
@@ -31,13 +39,40 @@ function RunDetail(): React.FunctionComponentElement<React.ReactNode> {
...
   const [dateNow, setDateNow] = useState(Date.now());
   const [isRunSelectDropdownOpen, setIsRunSelectDropdownOpen] = useState(false);
   const { runHash } = useParams<{ runHash: string }>();
+  const { url } = useRouteMatch();
+  const { pathname } = useLocation();
+  const [activeTab, setActiveTab] = useState(pathname);
 
-  const handleChange = (event: React.ChangeEvent<{}>, newValue: number) => {
-    setValue(newValue);
+  const tabs = [
```
[the remark this answers is not in the document]
Comment (roubkar) [reply aim/web/ui/src/pages/RunDetail/RunDetail.tsx:49]: nice idea! was thinking of the same, but we will also need to add lazy loading here,
I will add a TODO with you comment now for future if you don't mind

[the remark this answers is not in the document]
Comment (roubkar) [reply aim/web/ui/src/pages/RunDetail/RunDetail.tsx:49]: @Hamik25 please check the latest changes, have added the mapping of tab => component

**Review by Hamik25 — APPROVED** [review]

**Review by VkoHov — APPROVED** [review]

**Comment by VkoHov** [comment]
lgtm!!!

**Pull request merged**